### PR TITLE
Dockerhub

### PR DESCRIPTION
**Pull request by vvaltia**
https://github.com/CSCfi/rems/issues/2210

# Definition of Done / Review checklist

## Reviewability
- [ ] link to issue
- [ ] note if PR is on top of other PR
- [ ] note if related change in rems-deploy repo
- [ ] consider adding screenshots for ease of review

## API
- [ ] API is documented and shows up in Swagger UI
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible

## Documentation
- [ ] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page
- [ ] documentation _at least_ for config options (i.e. docs folder)
- [ ] ADR for major architectural decisions or experiments

## Different installations
- [ ] new configuration options added to rems-deploy repository
- [ ] instance specific translations (i.e. LBR kielivara)

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: .circleci/rahti-ci.sh [hunk 1]
```diff
@@ -18,13 +18,21 @@ else
...
 
 docker login -p $rahtitoken -u unused docker-registry.rahti.csc.fi
 docker push docker-registry.rahti.csc.fi/rems/rems:${tag1}
 docker push docker-registry.rahti.csc.fi/rems/rems:${tag2}
 
+if [ "${tag1}" == "release" ] ; then
+    docker login -u remspush -p ${dockerhub}
+    docker push rems:latest
+    docker push rems:${tag2}
+fi
```
Comment (opqdonut) [inline .circleci/rahti-ci.sh:34]: how about my idea of splitting the docker build & push out of the rahti deploy?

Comment (vvaltia) [reply .circleci/rahti-ci.sh]: Done

File: .circleci/config.yml [hunk 2]
```diff
@@ -118,21 +118,33 @@ jobs:
           paths:
             - target/uberjar/rems.jar
 
-  rahti-dev:
+  docker-circle:
```
Comment (opqdonut) [inline .circleci/config.yml:121]: would `docker-snapshot` be a better name for this?

**Review by opqdonut — APPROVED** [review]
looks good to me!

File: .circleci/config.yml (hunk 2 above)
Comment (vvaltia) [reply .circleci/config.yml]: Sure

**Review by opqdonut — APPROVED** [review]

**Pull request merged**